doc/api/test.md

```diff
@@ -1298,6 +1301,14 @@ changes:
       that specifies the index of the shard to run. This option is _required_.
     * `total` {number} is a positive integer that specifies the total number
       of shards to split the test files to. This option is _required_.
+  * `coverage` {boolean} Whether to collect code coverage or not.
+    **Default:** `false`.
+  * `coverageIncludeGlobs` {string|Array} Includes specific files in code coverage using a
```

Comment:
```suggestion
  * `coverageIncludePatterns` {string|Array} Includes specific files in code coverage using a
```

Reply:
This would change the wording which is used in all other places:
https://github.com/atlowChemi/node/blob/05f4e39cc27277686f9b1f45e5963aa4a5efd56d/lib/internal/test_runner/coverage.js#L498-L499
https://github.com/atlowChemi/node/blob/05f4e39cc27277686f9b1f45e5963aa4a5efd56d/lib/internal/test_runner/coverage.js#L498-L499
https://github.com/atlowChemi/node/blob/05f4e39cc27277686f9b1f45e5963aa4a5efd56d/lib/internal/test_runner/utils.js#L280-L289